Tentacle/Client.swift

```diff
@@ -294,103 +283,104 @@ public final class Client {
+    public func userWithLogin(_ login: String) -> SignalProducer<(Response, UserInfo), Error> {
+        return fetchOne(.userInfo(login: login))
     }
 
     /// Fetch the currently authenticated user
     public func authenticatedUser() -> SignalProducer<(Response, UserInfo), Error> {
-        return fetchOne(.AuthenticatedUser)
+        return fetchOne(.authenticatedUser)
     }
 
-    public func assignedIssues(page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
-        return fetchMany(.AssignedIssues, page: page, pageSize: perPage)
+    public func assignedIssues(_ page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
+        return fetchMany(.assignedIssues, page: page, pageSize: perPage)
     }
 
-    public func issuesInRepository(repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
+    public func issuesInRepository(_ repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
         precondition(repository.server == server)
-        return fetchMany(.IssuesInRepository(owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
+        return fetchMany(.issuesInRepository(owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
     }
 
     /// Fetch the comments posted on an issue
-    public func commentsOnIssue(issue: Int, repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Comment]), Error> {
+    public func commentsOnIssue(_ issue: Int, repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Comment]), Error> {
         precondition(repository.server == server)
-        return fetchMany(.CommentsOnIssue(number: issue, owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
+        return fetchMany(.commentsOnIssue(number: issue, owner: repository.owner, repository: repository.name), page: page, pageSize: perPage)
     }
 
     /// Fetch the authenticated user's repositories
-    public func repositories(page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [RepositoryInfo]), Error> {
-        return fetchMany(.Repositories, page: page, pageSize: perPage)
+    public func repositories(_ page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [RepositoryInfo]), Error> {
+        return fetchMany(.repositories, page: page, pageSize: perPage)
     }
 
     /// Fetch the repositories for a specific user
-    public func repositoriesForUser(user: String, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [RepositoryInfo]), Error> {
-        return fetchMany(.RepositoriesForUser(user: user), page: page, pageSize: perPage)
+    public func repositoriesForUser(_ user: String, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [RepositoryInfo]), Error> {
```

Comment:
Since the first parameter is `String`, this should be considered: https://swift.org/documentation/api-design-guidelines/#weak-type-information.